app/controllers/application_controller.rb

```diff
@@ -80,6 +80,10 @@ def failed_update_error(obj, obj_name)
   def failed_destroy_error(obj, obj_name)
     "#{_('Could not delete the %{o}.') % {o: obj_name}} #{errors_to_s(obj)}"
   end
+  
+  def success_message(obj_name, action)
```

Comment:
Since this is just a helper function to create text, perhaps this could live in a helper?  I noticed that we have a few other error/message creators in the application controller as well, but I think this type of function should be re-factored to be part of a helper.